test/test_torch.py

```diff
@@ -19146,25 +19146,29 @@ def _test_special_stacks(self, dim, at_least_dim, torch_fn, np_fn, device, dtype
     @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
     @dtypes(*(torch.testing.get_all_int_dtypes() + torch.testing.get_all_fp_dtypes(include_bfloat16=False) +
               torch.testing.get_all_complex_dtypes()))
-    def test_hstack(self, device, dtype):
-        self._test_special_stacks(1, 1, torch.hstack, np.hstack, device, dtype)
+    def test_hstack_column_stack(self, device, dtype):
+        ops = ((torch.hstack, np.hstack), (torch.column_stack, np.column_stack))
```

Comment:
This is a great way to extend this test.

Reply:
Yes, the hstack != column_stack as the hstack requires all input to have the same dim.
The latest commit extends this test method by adding test cases where input tensors are combinations of 1D and 2D tensors for the `torch.column_stack` function.